### PR TITLE
fix(curriculum):  Added description texts about spaces in step 154

### LINKED ISSUE #48087: Missing code block in step 154 of new JS rpg project
## the issue 

In the step, the hint says the following:

> You should add the string " Your " to `text.innerText`. Remember that spacing matters.

## the file

https://github.com/freeCodeCamp/freeCodeCamp/blob/6df93646db629a535c3ae193aee1c8e16d33feed/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa226207f33d3ad4c6f546.md#L27

<s>
### Additional context

If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution, followed by the one that is made first.

Happy contributing.
</s>

### PR DESCRIPTION
**Pull request by HatimZ**
Signed-off-by: HatimZ hatim.zahid1996@gmail.com

Checklist:

- [x] I have read freeCodeCamp's contribution guidelines.
- [x] My pull request has a descriptive title (not a vague title like Update index.md)
- [x]  My pull request targets the main branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.
Closes #48087

### CODE REVIEW

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please use the text decided in https://github.com/freeCodeCamp/freeCodeCamp/issues/48087#issuecomment-1285704305

**Comment by HatimZ** [comment]
I have made the respective changes according to https://github.com/freeCodeCamp/freeCodeCamp/issues/48087#issuecomment-1285704305.

Thanks.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa226207f33d3ad4c6f546.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ dashedName: step-154
 
 # --description--
 
-Use the `+=` operator to add ` Your [weapon] breaks.` to the end of `text.innerText`, replacing `weapon` with the last item in the `inventory` array. You can get this last item by using `inventory.pop()`, which will remove the last item in the array AND return it so it appears in your string.
+Use the `+=` operator to add ` Your [weapon] breaks.`, with a space in front of `Your`, to the end of `text.innerText`. Replace `[weapon]` with the last item in the `inventory` array using `inventory.pop()`, which will remove the last item in the array AND return it so it appears in your string.
```
Comment (ieahleen) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa226207f33d3ad4c6f546.md:10]:
```suggestion
Use the `+=` operator to add `Your [weapon] breaks.`, with a space in front of `Your`, to the end of `text.innerText`. Replace `[weapon]` with the last item in the `inventory` array using `inventory.pop()`, which will remove the last item in the array AND return it so it appears in your string.
```
This space can be removed, it's not going to be shown and it's going to confuse future contributors.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa226207f33d3ad4c6f546.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ You should use the `pop` method on the `inventory` array.
 assert.match(attack.toString(), /inventory\.pop\(\)/);
 ```
 
-You should add the string " Your " to `text.innerText`. Remember that spacing matters.
+You should add `Your`, with a space before and after it, to `text.innerText`. Remember that spacing matters.
```
Comment (ieahleen) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62aa226207f33d3ad4c6f546.md:27]:
```suggestion
You should add `Your`, with a space before and after it, to `text.innerText`.
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Comment by HatimZ** [comment]
Thanks!

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @HatimZ, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Comment by HatimZ** [comment]
Thanks @Sboonny . Its my pleasure. Sure I will be active in the future as well. Cheers!

**Pull request merged**